README.md

```diff
@@ -215,15 +215,16 @@ What this can do ?
 - Fetch the details of the user along with number of their followes and following
 - Fetch only the followers / following of a particular user
 
-Query Structure: `l.user(<username>,<followers count>,<following count>)`
+Query Structure: `l.user(<username>, <followers count>, <following count>)`
 
-`<username>` is a string, eg. `'loklak_app'`
+`<username>` is a string, e.g. `'loklak_app'`<br/>
 `<followers count>` and `<following count>` is a numeric or a string or `None`
 
-eg. `l.user('loklak_app')`
-eg. `l.user('loklak_app',1000)` - 1000 followers of `loklak_app`
-eg. `l.user('loklak_app',1000,1000)` - 1000 followers and following of `loklak_app`
-eg. `l.user('loklak_app',None,1000)` - 1000 following of `loklak_app`
+e.g.
+1. `l.user('loklak_app')`<br />
+2. `l.user('loklak_app',1000)` - 1000 followers of `loklak_app`<br />
+3. `l.user('loklak_app',1000,1000)` - 1000 followers and following of `loklak_app`<br />
+4. `l.user('loklak_app',None,1000)` - 1000 following of `loklak_app`<br />
```

Comment:
Please make the spaces consistent here too.

Reply:
Done!